specifications/api/swagger.yaml

```diff
@@ -267,6 +267,28 @@ x-walletName: &walletName
   minLength: 1
   example: Alan's Wallet
 
+x-poolMetadataSource: &poolMetadataSource
+  description: Pool metadata source. null -> no fetching, direct -> direct fetching, uri -> use SMASH server
+  oneOf:
+    - type: string
+      enum:
+        - null
+        - direct
+    - title: URI
+      type: string
+      format: uri
+      example: https://iohk.io
```

Comment:
Perhaps we can find the actual CF smash URL and use that as an example.

Reply:
I can't find any.

Reply:
Done